src/app/views/censusdata/create.html

```diff
@@ -0,0 +1,171 @@
+<div class="container">
+    <div class="well row" >
+        <h4>Census participants please be as complete as possible:</h4>
+        <ul>
+            <li>Review form guidelines prior to census.</li>
+            <li>Fill in/check box for all information in top section side one.</li>
+            <li>Always provide "number of adults and juveniles" data for species.</li>
+            <li>Always observe birds from at least 50 yards away.</li>
+        </ul>
+    </div>
+                
+    <form class="form-horizontal" role="form"> 
+        <div class="form-group panel panel-primary">
+            <div class="panel-heading">Section One</div>
+            <div class="panel-body">              
+            <div class="form-group">
+                <label class="control-label col-md-3" for="colonyCode">Colony Code:</label>
+                <div class="col-md-3">
+                <input type="text" class="form-control" id="colonyCode" placeholder="Enter Colony Code">
```

Comment:
Each one of the form elements needs to be bound to a model. So in the controller we need `$scope.survey` containing all of the model fields (`$scope.survey.colonyCode` for example), and here in the view we need this to be `<input ng-model="survey.colonyCode" ... >`